src/settingsmenu.cpp

```diff
@@ -38,7 +38,7 @@
 #include <algorithm>
 #include <assert.h>
 
-const Vec2i winSize(840, 356);
+const Vec2i winSize(840, 420);
```

Comment:
Each row has 64 pixels height (see `bWidgetH = 64` below). Adding 64 pixels to the window height for the new row.